frontend/widgets/OBSBasic_Transitions.cpp

```diff
@@ -803,12 +818,16 @@ QMenu *OBSBasic::CreatePerSceneTransitionMenu()
 			name = Str("None");
 
 		action = menu->addAction(QT_UTF8(name));
-		action->setProperty("transition_index", i);
+		action->setProperty("transition_uuid", QString::fromStdString(uuid));
 		action->setCheckable(true);
 		action->setChecked(match);
 
 		connect(action, &QAction::triggered, std::bind(setTransition, action));
-	}
+	};
+
+	addAction();
+	for (const auto &pair : transitions)
+		addAction(pair.first);
```

Comment:
```suggestion
	for (const auto &[name, transition] : transitions)
		addAction(name);
```
Using automatic decomposition makes it more semantically obvious what the code wants to achieve.

Reply:
Done